internal/cli/root/mongocli/builder_test.go

```diff
@@ -19,10 +19,12 @@ package mongocli
 import (
 	"bytes"
 	"fmt"
+	"strings"
 	"testing"
-	"time"
 
+	"github.com/Masterminds/semver/v3"
 	"github.com/golang/mock/gomock"
+	"github.com/google/go-github/v38/github"
```

Comment:
can we please update to v42?
```suggestion
	"github.com/google/go-github/v42/github"
```